TourGuide_Horde/29_30_HillsbradFoothills.lua

```diff
@@ -29,8 +76,8 @@ A Elixir of Pain (Part 2) |QID|502|
 T Helcular's Revenge |QID|552|
 A Helcular's Revenge (Part 2) |QID|553|
 
-C Helcular's Revenge (Part 2) |QID|553| |QO|Flame of Azel charged|
-C Helcular's Revenge (Part 2) |QID|553| |QO|Flame of Veraz charged|
+N Flame of Azel |QID|553| |Q|Helcular's Revenge (Part 2)| |QO|Flame of Azel charged|
+N Flame of Veraz |QID|553| |Q|Helcular's Revenge (Part 2)| |QO|Flame of Veraz charged|
```

Comment:
The (Part 2) portion is just for the player to get additional awareness. The actual quest title remains without that. Thus, we don't want that within our |Q| tags.